lumispy/signals/luminescence_spectrum.py

```diff
@@ -562,6 +586,53 @@ def px_to_nm_grating_solver(
         "\n", "\n\t"
     )
 
+    def centroid(self, slice=None, npeaks=1):
```

Comment:
If this isn't used, there is no need to create this parameter. It should be added when it is implemented.